### ISSUE TITLE
Results are set to 0 out of X when code in unit test hits specific project

### ISSUE THREAD

**Issue by Cjewett** [OpenCover/opencover#929]
Please provide the following information when submitting an issue. 

> Where appropriate replace the `[ ]` with a `[X]`

### My Framework

- [ ] .NET 2
- [ ] .NET 3.5
- [ ] .NET 4
- [ ] .NET 4.5
- [ ] .NET 4.6
- [X] .NET 4.6.1
- [ ] .NET 4.6.2
- [ ] .NET 4.7
- [ ] .NET 4.7.1
- [ ] .NET 4.7.2
- [ ] .NET Core 1.0.0
- [ ] .NET Core 2.0.0
- [ ] Something else

### My Environment

- [ ] Windows 7 or below (not truly supported due to EOL)
- [ ] Windows 8
- [ ] Windows 8.1
- [X] Windows 10
- [ ] Windows 10 IoT Core
- [ ] Windows Server 2012
- [ ] Windows Server 2012 R2
- [ ] Windows Server 2016

### I have already...

- [X] repeated the problem using the latest stable release of OpenCover.
- [X] reviewed the [usage guide](https://github.com/opencover/opencover/wiki/usage) and [usage document](https://github.com/opencover/opencover/blob/master/main/OpenCover.Documentation/Usage.pdf).
- [X] have looked at the opencover output xml file in an attempt to resolve the issue.
- [X] reviewed the [current issues](https://github.com/opencover/opencover/issues) to check that the issue isn't already known.

### My issue is related to (check only those which apply):

- [X] no coverage being recorded
- [ ] 32 or 64 bit support
- [ ] feature request

### Expected Behavior

OpenCover runs tests and generates code coverage output correctly.

### Actual Behavior

When running OpenCover against one of our Test projects I have noticed some odd behavior. This is an example project structure:

ExampleBusinessObjectsLibrary.csproj - 54 MB PDB file
ExampleUtilsLibrary.csproj - 2 MB PDB file
ExampleUnitTests.csproj

ExampleUnitTests.csproj references both ExampleUtilsLibrary.csproj and ExampleBusinessObjectsLibrary.csproj

If I write a unit test in ExampleUnitTest.csproj that calls a function in ExampleUtilsLibrary.csproj, and only hits code in ExampleUtilsLibrary.csproj the code coverage output is generated as expected. 

Now if I take that same unit test and hit any code in ExampleBusinessObjectsLibrary.csproj the code coverage output is not generated as expected. It sets all code coverage output to 0 out of X. 

ExampleUtilsLibrary.csproj is significantly smaller in size. 1/27th in size to be exact. 

Logs for failure: https://gist.github.com/Cjewett/2ff1bb1cc761e8368b47c3e6f02f0e47

### Steps to reproduce the problem:

I can't give reproduce code for this, and the steps are pretty simple. I'm just running the tool against one of our unit test projects.

Same command is always used.

C:\*\*\packages\OpenCover.4.7.922\tools\OpenCover.console.exe -register:user -target:"vstest.console.exe" -targetargs:"ExampleUnitTests.dll" -output:"coverage.xml"

- [X] reviewed the [usage guide](https://github.com/opencover/opencover/wiki/usage) and [usage document](https://github.com/opencover/opencover/blob/master/main/OpenCover.Documentation/Usage.pdf) again because we know you probably didn't do it the first time.
- [ ] sample code or link to repository/gist is available (support is much faster this way)

**Comment by Cjewett** [comment]
For anyone else running into this issue I managed to fix it by changing a couple files. 

https://github.com/OpenCover/opencover/blob/67980c8132b1d574beb1feaa7d269c9dca47b812/main/OpenCover.Profiler/ProfilerCommunication.cpp#L10 
Changed that to 600000. 

https://github.com/OpenCover/opencover/blob/67980c8132b1d574beb1feaa7d269c9dca47b812/main/OpenCover.Framework/CommandLineParser.cs#L248
For testing purposes I hardcoded this to 600000 as well. 

After changing those two lines and compiling locally I was able to run OpenCover against a 50 MB PDB and timeout was no longer an issue. 

@sawilde Does the above make sense? If so, it would be nice if the ProfilerCommunication.cpp value could be determined by user input instead of hardcoded. When I changed CommandLineParser.cs alone it did not work.
